sites/svelte.dev/vite.config.js

```diff
@@ -41,16 +41,32 @@ const config = {
 			'codemirror/addon/fold/xml-fold.js',
 			'codemirror/addon/fold/indent-fold.js',
 			'codemirror/addon/fold/markdown-fold.js',
-			'codemirror/addon/fold/comment-fold.js',
+			'codemirror/addon/fold/comment-fold.js'
 		],
-		exclude: ['@sveltejs/repl', '@sveltejs/site-kit'],
+		exclude: ['@sveltejs/repl', '@sveltejs/site-kit']
 	},
-	ssr: {noExternal: ['@sveltejs/site-kit']},
+	ssr: { noExternal: ['@sveltejs/site-kit'] },
 	server: {
 		fs: {
-			strict: false,
-		},
+			strict: false
+		}
 	},
+	// !HACK: Remove once vite 4.3 is out
```

Comment:
what's this about?

Reply:
Without this the site doesn't build at all. It entes an infinite loop until vercel times out at 45 minutes. Vite 4.3 will fix it, so this hack meanwhile